dandiapi/api/models/dandiset.py

```diff
@@ -62,6 +62,10 @@ def identifier(self) -> str:
     def embargoed(self) -> bool:
         return self.embargo_status == self.EmbargoStatus.EMBARGOED
 
+    @property
+    def unembargo_in_progress(self) -> bool:
```

Comment:
This might be better named as `unembargoing` but I don't have very strong feelings about it.

Reply:
Having a property `unembargoing` felt a bit vague, and this felt more clear.